### PR TITLE
added badges and alert

### PR DESCRIPTION
**Pull request by srijani07**
Alert:

image

Badges:

image

### CODE REVIEW

**Review by Ash-D23 — CHANGES_REQUESTED** [review]
You can add different color classes for alert, alert red success and warning.
Badge also can have different position like topright bottom right etc

**Review by md-amir-gauhar — APPROVED** [review]
Try adding different alerts with different colors for error, success, warning

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (rutvikpumak) [inline .vscode/settings.json:1]: no need of this file

File: Badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+.badge-wrapper {
+  position: relative;
+  display: inline-block;
+}
+
+.badge {
+  position: absolute;
+
+  color: white;
```
Comment (rutvikpumak) [inline Badges/badges.css:16]: use variable for color

**Comment by rutvikpumak** [comment]
image
We need to merge every branch to dev not main. Fix this ASAP

**Pull request closed without merging**